annif/backend/dummy.py

```diff
@@ -11,6 +11,10 @@ class DummyBackend(backend.AnnifLearningBackend):
     uri = 'http://example.org/dummy'
     label = 'dummy'
 
+    def default_params(self):
+        params = backend.AnnifBackend.DEFAULT_PARAMS.copy()
```

Comment:
`.copy()` can be dropped - this could just be `return backend.AnnifBackend.DEFAULT_PARAMS`